debian/rules

```diff
@@ -9,6 +9,8 @@ export DH_GOLANG_EXCLUDES=tests
 export DH_GOLANG_GO_GENERATE=1
 
 export PATH:=${PATH}:${CURDIR}
+# make sure that correct go version is found on trusty
```

Comment:
The golang-go package is the one that contains just the symlinks into /usr/lib and the man pages. golang-1.6* is what installs into /usr/lib/go-1.6. If we're getting an up-to-date version of go for trusty, why is it not following these same conventions?